app.go

```diff
@@ -183,6 +183,11 @@ type Config struct {
 	// Default: ""
 	ViewsLayout string `json:"views_layout"`
 
+	// PassLocalsToViews Enables passing of the locals set on a fiber.Ctx to the template engine
```

Comment:
We should probably mention that the Locals will be accessible to the template just like any other passed value, and that they will overwrite keys manually passed to `Render`.